publish/defs.bzl

```diff
@@ -144,32 +139,35 @@ def _artifact_bundle_impl(ctx):
     ctx.actions.run_shell(
         inputs = input_files,
         arguments = [file.path for file in input_files],
-        outputs = [out_checksums],
+        env = {
+            "BUNDLE_ROOT": bundle_root.path,
+            "BUNDLE_PREFIX": bundle_prefix,
+        },
+        outputs = [bundle_root],
         tools = [ctx.executable._sha256],
         command = """
         set -euo pipefail
 
-        out_checksums="{out}"
-        output=$(mktemp) # temporary file bc sha256 doesn't support writing to stdout (or /dev/stdout) directly
+        outdir="$BUNDLE_ROOT/$BUNDLE_PREFIX"
+
+        mkdir -p "$outdir"
+
+        out_checksums="$outdir/SHA256SUMS"
 
+        output=$(mktemp) # temporary file bc sha256 doesn't support writing to stdout (or /dev/stdout) directly
         for input in "$@"; do
-            if ! [[ $input =~ (\\.tar|\\.gz) ]]; then
-                echo "skipping non-archive file $input"
-                continue
-            fi
             {sha256} "$input" "$output"
             cat "$output" >> "$out_checksums"
             echo " $(basename $input)" >> "$out_checksums"
+            ln -s "$( realpath "$input" )" "$outdir/$(basename $input)"
         done
 
-        cat "$out_checksums"
```

Comment:
I decided to remove this from the logs because it creates noise and anyway is printed by `build-ic.sh`.